README.md

```diff
@@ -518,26 +518,22 @@ When these tests pass, your work is done.
 feel free to suggest a more _compact_ algorithm
 
 ```javascript
-var coins = [200, 100, 50, 20, 10, 5, 2, 1]
-function getChange(totalPayable, cashPaid) {
-    var change = [];
-    var length = coins.length;
-    var remaining = cashPaid - totalPayable;          // we reduce this below
-
-    for (var i = 0; i < length; i++) { // loop through array of notes & coins:
-        var coin = coins[i];
-
-        if(remaining/coin >= 1) { // check coin fits into the remaining amount
-            var times = Math.floor(remaining/coin);        // no partial coins
-
-            for(var j = 0; j < times; j++) {     // add coin to change x times
-                change.push(coin);
-                remaining = remaining - coin;  // subtract coin from remaining
-            }
-        }
+function getChange(cost, paid){
```

Comment:
Just read the original comment XD i will move it above the original one :-) And add some comments in the code so its a little more explanatory (being as this is a tutorial )